src/NzbDrone.Core/ImportLists/HttpImportListBase.cs

```diff
@@ -39,38 +39,44 @@ protected HttpImportListBase(IHttpClient httpClient, IImportListStatusService im
 
         public override ImportListFetchResult Fetch()
         {
-            var generator = GetRequestGenerator();
-            return FetchMovies(generator.GetMovies());
+            return FetchMovies(g => g.GetMovies());
         }
 
-        protected virtual ImportListFetchResult FetchMovies(ImportListPageableRequestChain pageableRequestChain, bool isRecent = false)
+        protected virtual ImportListFetchResult FetchMovies(Func<IImportListRequestGenerator, ImportListPageableRequestChain> pageableRequestChainSelector, bool isRecent = false)
         {
-            var movies = new List<ImportListMovie>();
+            var releases = new List<ImportListMovie>();
```

Comment:
Why releases? these are not releases like indexer, these are movies. if releases is used upstream it's likely a copy/paste error from the indexers loop